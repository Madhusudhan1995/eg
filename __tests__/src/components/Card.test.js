import React from "react";
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import { connect } from 'react-redux';
import CardItem from "../../../src/components/Card/index";


jest.mock('../../../src/components/Card/index', () => {
    return {
        default: jest.fn(),
    };
});

describe("CardItem Component", () => {
    let wrapper
    beforeEach(() => {
        wrapper = shallow(<CardItem />)
    })
    it('CardItem should render correctly', () => {
        const tree = renderer.create(wrapper).toJSON();
        expect(tree).toMatchSnapshot()
    });
});

