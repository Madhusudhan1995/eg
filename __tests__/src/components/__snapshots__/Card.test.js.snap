// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardItem Component CardItem should render correctly 1`] = `null`;
